binr/r2r/r2r.c

```diff
@@ -631,3 +646,83 @@ static void print_log(R2RState *state, ut64 prev_completed, ut64 prev_paths_comp
 		printf ("\n");
 	}
 }
+
+static void interact(R2RState *state) {
+	void **it;
+	RPVector failed_results;
+	r_pvector_init (&failed_results, NULL);
+	r_pvector_foreach (&state->results, it) {
+		R2RTestResultInfo *result = *it;
+		if (result->result == R2R_TEST_RESULT_FAILED) {
+			r_pvector_push (&failed_results, result);
+		}
+	}
+	if (r_pvector_empty (&failed_results)) {
+		goto beach;
+	}
+
+	printf ("\n");
+	printf ("#####################\n");
+	printf (" %"PFMT64u" failed test(s) \xf0\x9f\x9a\xa8\n", (ut64)r_pvector_len (&failed_results));
+
+	r_pvector_foreach (&failed_results, it) {
+		R2RTestResultInfo *result = *it;
+		if (result->test->type != R2R_TEST_TYPE_CMD) {
+			// TODO: other types of tests
+			continue;
+		}
+
+		printf ("#####################\n\n");
+		print_result_diff (&state->run_config, result);
+inval:
+		printf ("Wat do?    "
+				"(f)ix \xe2\x9c\x85\xef\xb8\x8f\xef\xb8\x8f\xef\xb8\x8f    "
+				"(i)gnore \xf0\x9f\x99\x88    "
+				"(b)roken \xe2\x98\xa0\xef\xb8\x8f\xef\xb8\x8f\xef\xb8\x8f    "
+				"(c)ommands \xe2\x8c\xa8\xef\xb8\x8f    "
+				"(q)uit \xf0\x9f\x9a\xaa\n");
+		printf ("> ");
+		char buf[0x30];
+		if (!fgets (buf, sizeof (buf), stdin)) {
+			break;
+		}
+		if (strlen (buf) != 2) {
+			goto inval;
+		}
+		switch (buf[0]) {
+		case 'f':
+			interact_fix (result, &failed_results);
+			break;
+		case 'i':
```

Comment:
run cfg.editor (EDITOR|vi) with a file containing the test in the right line (or maybe just do like r2r-i that deserializes and serializes the test files

Reply:
We have no `cfg.editor` in r2r because we have no core. I don't understand what you mean by the rest.